WooCommerce/Classes/POS/Utils/PreviewHelpers.swift

```diff
@@ -18,7 +18,11 @@ struct POSProductPreview: POSItem {
 }
 
 final class POSItemProviderPreview: POSItemProvider {
-    func providePointOfSaleItems() -> [POSItem] {
+    func providePointOfSaleItemsFromNetwork() async throws -> [Yosemite.POSItem] {
+        []
+    }
+
+    func providePointOfSaleItemsFromStorage() -> [POSItem] {
```

Comment:
We could remove this method at the moment, since we're just fetching from the remote. I've kept it around for faster testing but it's redundant now. Thoughts?